internal/ip_neighbor/ip_neighbor.go

```diff
@@ -0,0 +1,38 @@
+package ip_neighbor
+
+import (
+	"net"
+	"net/netip"
+
+	nl "github.com/vishvananda/netlink"
+)
+
+type Neighbors struct {
```

Comment:
imo should be named `Neighbor` as this struct represents a single Neighbor entry

Reply:
fixed